client/webserver/site/src/js/forms.ts

```diff
@@ -1058,7 +1058,7 @@ export class WalletWaitForm {
     Doc.show(wallet.synced ? page.syncCheck : wallet.syncProgress >= 1 ? page.syncSpinner : page.syncUncheck)
     Doc.show(wallet.balance.available > fee.amount ? page.balCheck : page.balUncheck)
 
-    page.progress.textContent = String(Math.round(wallet.syncProgress * 100))
+    page.progress.textContent = String((wallet.syncProgress * 100).toFixed(1))
```

Comment:
Nice.  Is there any remaining purpose of the `String` *object* constructor here instead of the string primitive that we get from `toFixed`?

Reply:
Not that I'm aware of, I just didn't notice `toFixed` returned **string**, removed this conversion.